### PR TITLE
Update deps and fix schema

### PR DESCRIPTION
**Pull request by davedoesdev**
Includes fix for #40 

All the commands in `package.json` now work, tests all pass and doc generation works.

I've added the generated `doc` directory and the browserified `dist/EBML.js` file.

### CODE REVIEW

**Comment by onthegit** [comment]
It appears that you have incorporated my changes from https://github.com/legokichi/ts-ebml/pull/41 into this one so I will close mine, including those of TimecodeScale.

@legokichi gentle ping  :)

**Comment by davedoesdev** [comment]
I've also added running the tests using Playwright, and moved from Travis to Github Actions.

File: package.json [hunk 1]
```diff
@@ -40,30 +41,45 @@
...
-  "typings": "./lib/index.d.ts"
+  "typings": "./lib/index.d.ts",
+  "browserify": {
+    "transform": [
+      "aliasify"
+    ]
+  },
+  "aliasify": {
+    "aliases": {
+      "ebml": "ebml/lib/ebml.js"
```
Comment (thatsmydoing) [inline package.json:82]: I just wanted to point out that this is important for consumers of `ts-ebml` too. The dependency `ebml` 3.0.0 has a `browser` entry in its package.json that points at an IIFE bundle which is not what bundlers expect so it has to be aliased to `ebml/lib/ebml.js` or `ebml/lib/ebml.esm.js`

**Comment by buddhikajay** [comment]
Nice work @davedoesdev & @thatsmydoing 
@legokichi, any plans to merge this?

**Comment by vhmth** [comment]
@davedoesdev thank you so much for your hard work here! 🙏🏾 

Given this fixes a pretty critical [licensing issue](https://github.com/legokichi/ts-ebml/issues/40), is there a timeline on merging? If you're under water, happy to take on just the updating of where we pull the spec in a separate PR. Lmk!

**Comment by legokichi** [comment]
@davedoesdev 
Sorry for the long delay.
I finally got some time off and will try to merge this commits in https://github.com/legokichi/ts-ebml/pull/46 .
Thanks for your great contribution!

**Pull request closed without merging**